pkg/trace/local_tracer.go

```diff
@@ -142,26 +145,26 @@ func (lc *LocalClient) saveEventToFile(event Event[Entry]) error {
 }
 
 // draincanal takes a variadic number of channels of Event pointers and drains them into files.
-func (lc *LocalClient) draincanal() {
+func (lt *LocalTracer) draincanal() {
```

Comment:
[question] what's a Canal? Why does this method not use camelCase?

Reply:
fixed, just a silly error